server/controllers/stories.controller.js

```diff
@@ -0,0 +1,65 @@
+"use strict";
+const {StoriesService} = require('../services');
+
+class StoriesControler {
+  constructor() {}
+
+  async getAllStories(req, res) {
+    const data = await StoriesService.findAllStories();
+    return res.status(data.status).send(data.message);
+  }
+
+  async getStroyById(req, res) {
+    const id = req.params.id;
+    if(!id){
+      return res.status(400).send({ message: 'Invalid request parameter!' });
+    }
+    const data = await StoriesService.findStoryById(id);
+    return res.status(data.status).send(data.message);
+  }
+
+  async getStroiesByAuthor(req, res) {
+    const authorId = req.params.authorId;
+    if(!authorId){
+      return res.status(400).send({ message: 'Invalid request parameter!' });
+    }
+    const data = await StoriesService.findStoriesByAuthor(authorId);
+    return res.status(data.status).send(data.message);
+  }
+
+  async createStory(req, res) {
+    const {title, description} = await req.body;
+    if (!title || !description) {
+      return res.status(400).send({ message: 'Invalid request body' });
+    }
+    const {status,message} = await StoriesService.createStroy(title, description);
+    if(status.toString().startsWith('4')){
+        return res.status(status).send(message);
+    }
```

Comment:
in both cases, if the status is 4xx series or other you are executing the same line 'res.status(status).send(message)'